Tests/scripts/jira_issues.py

```diff
@@ -88,3 +96,33 @@ def jira_server_information(jira_server: JIRA):
     logging.info("Jira server information:")
     for key, value in jira_server_info.items():
         logging.info(f"\t{key}: {value}")
+
+
+def jira_search_all_by_query(jira_server: JIRA,
```

Comment:
that's the majority of the change, to perform less operations against jira, as they're ratelimiting the API on the new instance.